shell/platform/embedder/BUILD.gn

```diff
@@ -87,6 +87,12 @@ embedder_source_set("embedder_with_symbol_prefix") {
   public_configs = [ ":embedder_prefix_config" ]
 }
 
+source_set("embedder_headers") {
+  sources = [
+    "embedder.h",
+  ]
```

Comment:
You should probably add `public_configs += [ "//flutter:config" ]` to this, just in case there were ever some global config (e.g., a define) that the header used.